bin/install.sh

```diff
@@ -284,11 +293,19 @@ case "\${INPUT}" in
                 if [ ! -z "\${CODECOV_TOKEN}" ]; then
                     CODECOV_EXE="\${CODECOV_EXE} -t \${CODECOV_TOKEN}"
                 fi
-                \${CODECOV_EXE} \
-                    -R "${CIRRUS_WORKING_DIR}" \
-                    --file lcov.info \
-                    --source "github.com/ararslan/CirrusCI.jl" \
-                    --verbose
+                if [ -z "\${JULIA_PROJECT_SUBDIR}" ]; then
+                    \${CODECOV_EXE} \
+                        -R "${CIRRUS_WORKING_DIR}" \
+                        --file lcov.info \
+                        --source "github.com/ararslan/CirrusCI.jl" \
+                        --verbose
+                else
+                    \${CODECOV_EXE} \
+                        -R "${CIRRUS_WORKING_DIR}/${JULIA_PROJECT_SUBDIR}" \
+                        --file lcov.info \
+                        --source "github.com/ararslan/CirrusCI.jl" \
+                        --verbose
+                fi
```

Comment:
The same comment applies here:
```suggestion
                \${CODECOV_EXE} \
                    -R "${CIRRUS_WORKING_DIR}/${JULIA_PROJECT_SUBDIR}" \
                    --file lcov.info \
                    --source "github.com/ararslan/CirrusCI.jl" \
                    --verbose
```